src/components/sidebar.js

```diff
@@ -8,34 +8,20 @@ import { config } from "./config";
 import { configGuides } from "./config-guides";
 import Icon from "./icon";
 
-class Sidebar extends React.Component {
-  renderTransformedToc(siblings, targetLocation) {
-    const md = MarkdownIt();
-
-    return (
-      <ul className="Sidebar-toc">
-        {
-          siblings.map((sibling, id) => {
-            if (Array.isArray(sibling)) {
-              return (
-                <li className="Sidebar-toc-item" key={id}>
-                  {this.renderTransformedToc(sibling, targetLocation)}
-                </li>
-              );
-            }
+class SidebarListItem extends React.Component {
```

Comment:
I think we're generally one component / file, but these two are so closely coupled I don't mind them hanging out together.